WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsOnboardingErrorButtonViewModel.swift

```diff
@@ -0,0 +1,28 @@
+import Foundation
+import Yosemite
+
+struct InPersonPaymentsOnboardingErrorButtonViewModel {
+    let text: String
+
+    private let analyticReason: String?
```

Comment:
As we're dealing with an onboarding error when this viewModel is called, should we make the `analyticReason` non-optional so we're sure to track them?